src/main/java/mate/academy/spring/validator/RepeatPasswordValidator.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class RepeatPasswordValidator
+        implements ConstraintValidator<PasswordRepeated, UserRequestDto> {
+    private static final Pattern EMAIL_PATTERN
+            = Pattern.compile("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$");
+
+    @Override
+    public boolean isValid(UserRequestDto value, ConstraintValidatorContext context) {
+        return value.getPassword().equals(value.getRepeatPassword());
```

Comment:
add password null check because we can get an NPE here, since the order of the constraints validations is not guaranteed